src/main/java/graphql/servlet/AbstractGraphQLHttpServlet.java

```diff
@@ -109,7 +109,28 @@ public AbstractGraphQLHttpServlet(List<GraphQLServletListener> listeners, boolea
                                     Collections::singletonList,
                                     (l1, l2) -> Stream.concat(l1.stream(), l2.stream()).collect(Collectors.toList())));
 
-                    if (fileItems.containsKey("graphql")) {
+                    if(fileItems.containsKey("operations")) {
+                        final Optional<Part> queryItem = getFileItem(fileItems, "operations");
+                        if (queryItem.isPresent()) {
```

Comment:
We're starting to get a lot of duplication here. This `if` block seems to be an exact duplicate of the one below in the `fileItems.containsKey("graphql")` block. Would you mind refactoring a bit by extracting that block in a separate method and replacing both blocks with a call to that method?

Reply:
Yes, no problem. I'm also adding built-in support for the `map` parameter that the multipart request spec uses, as it's not possible to layer in that behavior as a library consumer. I'll include this refactoring in my next update.